streetcrm/admin.py

```diff
@@ -552,8 +551,24 @@ def advanced_search_do(self, request, form, export):
                                "result_count": count_set['result_count']
             }
         else:
+            search_params = []
+            for key, value in data.items():
+                if isinstance(value, datetime):
+                    search_params.append('{} = {}'.format(key, value.strftime('%Y-%m-%d')))
+                elif isinstance(value, str) and len(value):
+                    search_params.append('{} = {}'.format(key, value))
+                elif isinstance(value, bool) and value:
+                    search_params.append(key)
+                elif hasattr(value, '__iter__') and len(value):
+                    search_params.append('{} = {}'.format(key, ','.join([str(v) for v in value])))
+                elif hasattr(value, 'pk'):
+                    search_params.append('{} = {}'.format(key, str(value)))
+
+            search_header = ', '.join(search_params)
+
             results_package = {"form": form,
                                "search_results": results,
+                               "search_header": search_header,
```

Comment:
Personally I don't have a strong opinion on whether a separate method is called for -- really, could go either way.  So unless @cecilia-donnelly does have a strong opinion, I'd say just do whichever way feels most right (probably easier to know once you see the updated code in place anyway).